src/less/components/button.less

```diff
@@ -1,279 +1,5 @@
-
-    &[active],
-    &:active,
-    &.active,
-    &[disabled],
-    fieldset[disabled] & {
-        background-color: transparent;
-        box-shadow: none;
-    }
-
-    &:hover,
-    &:active {
-        box-shadow: none;
-        border-color: @color;
-    }
-
-    &:focus {
-        outline: 0;
-        box-shadow: 0 0 0 0.1rem @focus-shadow;
-        border-color: @color;
-    }
-
-    &:hover,
-    &:focus {
-        background-color: transparent;
-    }
-
-    &[disabled],
-    fieldset[disabled] & {
-        color: @disabled-color;
-        text-decoration: none;
-        border: none;
-    }
-
-    span {
-        text-decoration: none;
-    }
-
-    span:only-child {
-        text-decoration: underline;
-    }
-}
-
-.button-ghost(@color) {
-    color: @white;
-    background: transparent;
-    border: 1px solid @white;
-
-    &:focus,
-    &.focus,
-    &:hover,
-    &:active {
-        .button-active-state(@white, @color);
-    }
-}
-
-.button-link-variant(@color) {
-    color: @color;
-
-    &:hover,
-    &:focus {
-        color: darken(@color, 10%);
-    }
-}
-
-.button-active-state(@border) {
-    outline: 2px solid @border;
-}
-
-.button-outline-variant-active-state(@background; @border; @text) {
-    background-color: @background;
-    outline: 2px solid @border;
-    color: @text;
-}
-
-// Button sizes
-.button-size(@padding-horizontal; @height; @font-size; @line-height; @border-radius) {
-    padding: 0 @padding-horizontal;
-    font-size: @font-size;
-    line-height: @line-height;
-    border-radius: @border-radius;
-    height: @height;
-}
-
-.button-icon-size(@padding-horizontal; @height; @width; @font-size; @line-height; @border-radius) {
-    padding: 0 @padding-horizontal;
-    font-size: @font-size;
-    line-height: @line-height;
-    border-radius: @border-radius;
-    height: @height;
-    width: @width;
-}
-
-.btn-loader-variant(@max-height; @min-height; @width; @margin; @font-size; @btn-height) {
-    max-height: @max-height;
-    display: inline-block;
-    position: absolute;
-    top: calc(~"50% - @{btn-height}");
-    left: 0;
-    margin-left: auto;
-    margin-right: auto;
-    width: 100%;
-
-    .loader-icon {
-        bottom: (@font-size / 2) - (@font-size / 8);
-        width: (@width * 3) + (@margin * 2);
-        height: @max-height / 2;
-
-        li {
-            width: @width;
-            height: @min-height;
-            margin-right: @margin;
-            margin-top: (@max-height - @min-height) / 2;
-        }
-
-        li:first-child {
-            margin-right: 0;
-        }
-    }
-}
-
-@btn-font-size:                             1.125rem;
-@btn-min-width:                             112px;
-
-@btn-font-size-sm:                          1.6rem;
-@btn-font-size-xs:                          1.2rem;
-
-@btn-padding-horizontal:                    1rem;
-@btn-default-height:                        3rem;
-@btn-default-width:                         auto;
-
-.button-group {
```

Comment:
we're removing `.btn-group` ?
I'm not sure what was its point, but I just want to double-check this is on purpose 😉